make_datasets.py

```diff
@@ -14,8 +14,15 @@ def gen_regression(args):
                            bias=rs.normal(0, 3),
                            random_state=rs)
 
-    np.save(args.filex, X)
```

Comment:
I think I missed this last time, but at the top of this file, there should be the usual copyright header

```python
# Copyright (C) 2018-2020 Intel Corporation
# 
# SPDX-License-Identifier: MIT
```

Part of the code for some of these files dates back to 2017, so that's why I have been adding earlier years as well.

Reply:
Header was added